suites/quincy/rgw/tier-1_rgw_upgrade_4x_to_6x_latest.yaml

```diff
@@ -0,0 +1,411 @@
+#############################################################################
+# - Automation support for upgrade from RHCS4x to RHCS6x cluster with RGW tests
+# with dashboard enabled
+#
+# Test steps:
+# ---------------------------------------------------------------------
+# - Deploy bare-metal Nautilus Ceph cluster using CDN RPMS with lvm osd scenario
+# - Run RGW tests
+# - Convert the cluster to containerized using ceph-ansible
+# - Upgrade to quincy using ceph-ansible
+# - Run RGW tests after upgrade
+#
+#############################################################################
+tests:
+- test:
+    name: install ceph pre-requisites
+    module: install_prereq.py
+    abort-on-fail: true
+
+- test:
+    name: ceph ansible install rhcs 4.x from cdn
+    polarion-id: CEPH-83571467
+    module: test_ansible.py
+    config:
+      use_cdn: true
+      build: '4.x'
+      ansi_config:
+        ceph_origin: repository
+        ceph_repository: rhcs
+        ceph_repository_type: cdn
+        ceph_rhcs_version: 4
+        ceph_stable_release: nautilus
```

Comment:
This key/value is not required and has no impact since we are setting `ceph_repository: rhcs` hence it is necessary to set `ceph_rhcs_version`

Refer: https://docs.ceph.com/projects/ceph-ansible/en/stable-4.0.25/installation/methods.html

Reply:
done